other/bootstrap_daemon/src/log.c

```diff
@@ -85,12 +88,13 @@ void log_syslog(LOG_LEVEL level, const char *format, va_list args)
 FILE *level_stdout(LOG_LEVEL level)
 {
     switch (level) {
-        case LOG_LEVEL_INFO:
-            return stdout;
-
```

Comment:
Your changes are not similar. In the syslog function you just added a default case, but in stdout you additionally moved `LOG_LEVEL_INFO` case. I would appreciate if you don't move the `LOG_LEVEL_INFO` case from where it originally was.